pom.xml

```diff
@@ -0,0 +1,51 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud.tools</groupId>
+  <artifactId>gcp-java-tools-lib</artifactId>
+  <version>0.2-SNAPSHOT</version>
+
+  <name>Google Cloud Java management client library</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <appengine.target.version>1.9.32</appengine.target.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-tools-sdk</artifactId>
+      <version>${appengine.target.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
```

Comment:
It will contain its dependencies, but the problem is that those deps can conflict with IJ SDK provided deps (or from deps of your other plugin dependencies).  Best to stick to the same version of Guava as IJ for now as I think your use of it doesn't depend on 19.